scielomanager/editorialmanager/models.py

```diff
@@ -35,7 +35,7 @@ class EditorialMember(models.Model):
     link_cv = models.URLField(_('Link CV'), null=True, blank=True)
     city = models.CharField(_('City'), max_length=256, null=True, blank=True)
     state = models.CharField(_('State'), max_length=256, null=True, blank=True)
-    country = CountryField(_('Country'), default='')
+    country = CountryField(_('Country'))
```

Comment:
O campo ``role`` esta da mesma forma, se isso for impedimento irei remover essa alteração e agente abre um PR somente para isso, OK?